core/controllers/classroom_test.py

```diff
@@ -274,6 +281,81 @@ def test_not_able_to_get_classroom_data_when_user_is_not_admin(
             feconf.CLASSROOM_ADMIN_DATA_HANDLER_URL, expected_status_int=401)
         self.logout()
 
+    def test_get_unused_topics(self) -> None:
```

Comment:
Can you please break this test into multiple smaller tests with the following names:
* `test_returns_newly_added_unused_topics`
* `test_does_not_return_deleted_topic`